packages/kbn-test/src/jest/junit_reporter/junit_reporter.ts

```diff
@@ -71,6 +72,7 @@ export default class JestJUnitReporter extends BaseReporter {
       tests: results.numTotalTests,
       failures: results.numFailedTests,
       skipped: results.numPendingTests,
+      'command-line': prettifyCommandLine(process.argv),
```

Comment:
seems to overlap with https://github.com/elastic/kibana/pull/187060.  do we need the whole string?  is the config sufficient?

Reply:
Thanks for connecting these!
Our ultimate goal is to have an easy-to-grab command for the developers to replay the tests locally. If you think this is too much extra, I can try to work around, and get this only from the config. 
However, in this PR, the solution works the same way for all 3 types of tests (jest/jest_integration/ftr) - in #187060 the config is only extracted for FTR. Maybe we could try adopting the other way around?

Reply:
We've talked about it on the QA side and we think providing the full command line makes it even easier to run/debug locally, so we've closed the config listing PR in favor of this one.

Reply:
Thanks!